content/c/concepts/linked-list/linked-list.md

```diff
@@ -0,0 +1,329 @@
+---
+Title: 'Linked List'
+Description: 'A data structure that contains a collection of nodes that together with the help of pointers, display a sequence.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Singly linked lists'
+  - 'Doubly linked lists'  
+  - 'Circular linked lists'
+  - 'Circular doubly linked lists'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A **`Linked List`** is a data structure that contains a collection of `nodes` that together with the help of pointers, display a sequence. There are several list types, we will cover four of the most common with examples in the [C language](https://www.codecademy.com/resources/docs/c).
+
+## Syntax
```

Comment:
The Syntax section seems to be superfluous given the below examples. Usually a Syntax section really only applies to an illustration of using a built-in function or method, where this is more an illustration of linked lists in C.

Reply:
@SSwiniarski For this one I'm not quite sure how to turn this into a simple syntax without removing how it works. What do you recommend or have as a rough example to replace?